src/panels/config/lovelace/dashboards/ha-config-lovelace-dashboards.ts

```diff
@@ -59,8 +60,41 @@ export class HaConfigLovelaceDashboards extends LitElement {
           sortable: true,
           filterable: true,
           direction: "asc",
+          template: (title, dashboard: any) => {
+            const titleTemplate = html`
+              ${title}
+              ${dashboard.default
+                ? html`
+                    <ha-icon
+                      style="padding-left: 10px;"
+                      icon="hass:check-circle-outline"
+                    ></ha-icon>
+                    <paper-tooltip>
+                      This is the default dashdoard.
+                    </paper-tooltip>
+                  `
+                : ""}
+            `;
+            return narrow
+              ? html`
+                  ${titleTemplate}
+                  <div class="secondary">
+                    ${this.hass.localize(
+                      `ui.panel.config.lovelace.dashboards.conf_mode.${dashboard.mode}`
```

Comment:
Is it possible this doesn't return a value that we need `|| dashboard.mode` ?

Reply:
Not at the moment no